rrn.py

```diff
@@ -70,6 +70,22 @@ def _validate_hash(rrn: str) -> bool:
         return True
 
 
+def _is_valid_domestic_rrn(rrn: str) -> bool:
+    return (
+        rrn.isdigit() and
+        _validate_birth(rrn) and
+        _validate_location(rrn) and
+        _validate_hash(rrn)
+    )
+
+
+def _is_valid_foreign_rrn(rrn: str) -> bool:
+    return (
```

Comment:
별건 아닌데 아래처럼하면 어떨까요? 향후에 `location` 검사랑 `hash` 검사가 들어갈 가능성때문에 이렇게 두신건가요?

```suggestion
    return rrn.isdigit() and _validate_birth(rrn)
```

Reply:
@JungWinter 네 맞습니다. 1) 추후에 추가될 수 있다는 판단과 2) 내국인 유효성 검사 코드와 구조를 비슷하게 가져가는 게 좋을 것 같았는데요. 1) 그냥 더 깔끔한 코드를 유지하자 2) Suggested change를 이용해 commit을 추가해보고 싶은 마음이 커서 바로 적용했습니다.

Reply:
억 ㅋㅋㅋ 제안해주신 코드에 뭔가 스타일 오류가 있나보네요 `IndentationError: unexpected indent`

Reply:
아하 정말 저 line의 코드만 수정이 되었네요 👀

```python
return rrn.isdigit() and _validate_birth(rrn)
        rrn.isdigit() and
        _validate_birth(rrn)
    )
```

Reply:
아직 beta라 ㅠ_ㅠ 저도 한번 써본건데 딱 1줄만 적용되네요